doc/configcommands.dsv

```diff
@@ -73,6 +73,7 @@ oldreader-min-items|<number>|20|This variable sets the number of articles that a
 oldreader-password|<password>|""|This variable sets your The Old Reader password for The Old Reader support. Double quotes should be escaped, i.e. you should write +{backslash}"+ instead of +"+.|oldreader-password "here_goesAquote:{backslash}""
 oldreader-passwordfile|<path-to-file|""|A more secure alternative to the above, by storing your password elsewhere in your system.|oldreader-passwordfile "path-to-file"
 oldreader-show-special-feeds|[yes/no]|yes|If this is set, then "special feeds" like "People you follow" (articles shared by people you follow), "Starred items" (your starred articles) and "Shared items" (your shared articles) appear in your subscription list.|oldreader-show-special-feeds "no"
+openbrowser-and-mark-jumps-to-next-unread|[yes/no]|no|If set to yes, jump to the next unread item when an item is opened to the browser and marked as read(Default: O).|openbrowser-and-mark-jumps-to-next-unread yes
```

Comment:
1. An item is opened *in* the browser, not *to* it.
2. We already have the default value in the third column, so no need to put "(Default: 0)" at the end of the description.